assets/installation.md

```diff
@@ -13,11 +13,15 @@ Below is a listing of installation for each platform.
 We have not yet restored building the latest versions for Windows.
 Please see the [Vega Strike 0.5.1](https://sourceforge.net/projects/vegastrike/files/vegastrike/0.5.1/) for a release to use.
 
+NOTE: Version 0.8.0 has reestored building on Windows. Pre-built binaries are not yet being provided at this time.
+
 ## Mac OS X
 
 We have not yet restored building the latest versions for Mac OS X.
 Please see the [Vega Strike 0.5.0](https://sourceforge.net/projects/vegastrike/files/vegastrike/0.5.0/) for a release to use.
 
+NOTE: Version 0.8.0 has reestored building on Mac OS X. Pre-built binaries are not yet being provided at this time.
```

Comment:
`restored` not `reestored`, as above

Reply:
Must remember that Github does not do auto-correct ;-)